CHANGELOG.md

```diff
@@ -16,7 +16,7 @@
 
 ### Bug fixes
```

Comment:
I understand these come from PRs with typos, but my belief is that it's better for a person reading a changelog to be able to find things in it or understand it. If they want to see the typo, they can click the link to the PR. And anyone trying to find a changelog entry for a PR based on the PR can search for the PR (`/pull/8551` or similar) instead of the prose.

That said, I'm happy to drop changes to this file or others.